src/sql/parts/connection/common/connectionActions.ts

```diff
@@ -161,8 +163,14 @@ export class GetCurrentConnectionStringAction extends Action {
 				let includePassword = false;
 				let connectionProfile = this._connectionManagementService.getConnectionProfile(activeInput.uri);
 				this._connectionManagementService.getConnectionString(connectionProfile.id, includePassword).then(result => {
+
+					//Copy to clipboard
+					this._clipboardService.writeText(result);
+
 					let message = result
-						? result
+						? nls.localize('connectionAction.copyConnectionString', 'Connection string to ') +
```

Comment:
I think we should copy to the clipboard in addition to showing in the notification window.  Can we update the PR to add the change on line 168, but to continue to display the connection string to the user?